deploy/01_d2dBal_and_balDepositor.js

```diff
@@ -20,7 +20,7 @@ const deployFunction = async ({ getNamedAccounts, deployments }) => {
   });
 
   // Owner of D2DBal should be BalDepositor
-  await execute('D2DBal', { from: root, log: true }, 'transferOwnership', balDepositor)
+  await execute('D2DBal', { from: root, log: true, gasLimit: 6000000 }, 'transferOwnership', balDepositor)
```

Comment:
Why do we need to set the gasLimit strictly?

Reply:
When there is a big network load, it fails to determine gasLimit automatically, and doesn't execute tx, throws an error